### ISSUE TITLE
Examples of input and attribute files

### ISSUE THREAD

**Issue by jenkelly10** [rtewhey/MPRA_Tag_Analysis#1]
Hi Ryan, 

Thanks for all of your tremendous recent work on the MPRA Oligo-Tag pipeline.  Your example folder is especially helpful.  Would it be possible for you to include examples of your input (Geuv_90K_sep_raw.counts.final) and attribute (master.geuvadis.probes.AsiSIremoved.out.class) files that are used in your DESeq_overexpression_BF_CombinedAll.r script?  Just want to make sure I have the correct file formats and files to put into the DESeq r script after steps 3 (assigning oligos to barcodes) and 4 (building count matrix) of the MPRA_Tag_Analysis.  Thank you!

**Comment by rtewhey** [comment]
Absolutely, I have neglected the tag analysis scripts as I spend time polishing the oligo/tag pipeline. We have an updated pipeline for analyzing the plasmid/cDNA sequencing that I will try and get online soon.
